.travis.yml

```diff
@@ -2,15 +2,11 @@ language: php
 
 php:
   - 7.4
-  - nightly
+  - 8.0snapshot
 
 before_script:
   - phpenv config-rm xdebug.ini || true
   - composer update --no-interaction
 
 script:
   - vendor/bin/phpunit
-
```

Comment:
We should allow `8.0snapshot` to fail since 8.0 has not been released yet.

Reply:
changed :)